test-support/src/main/scala/com/datastax/spark/connector/ccm/CcmConfig.scala

```diff
@@ -116,16 +117,16 @@ object CcmConfig {
   val DEFAULT_SERVER_LOCALHOST_KEYSTORE_PATH: String = "/server_localhost.keystore"
 
   // DSE versions
-  val V6_8_5: Version = Version.parse("6.8.5")
-  val V6_8_3: Version = Version.parse("6.8.3")
-  val V6_8_0: Version = Version.parse("6.8.0")
-  val V6_7_0: Version = Version.parse("6.7.0")
-  val V6_0_0: Version = Version.parse("6.0.0")
-  val V5_1_0: Version = Version.parse("5.1.0")
-  val V5_0_0: Version = Version.parse("5.0.0")
+  val DSE_V6_8_5: Version = Version.parse("6.8.5")
+  val DSE_V6_8_3: Version = Version.parse("6.8.3")
+  val DSE_V6_7_0: Version = Version.parse("6.7.0")
+  val DSE_V6_0_0: Version = Version.parse("6.0.0")
+  val DSE_V5_1_0: Version = Version.parse("5.1.0")
+  val DSE_V5_0_0: Version = Version.parse("5.0.0")
 
   // C* versions
-  val V4_0_0: Version = Version.parse("4.0.0")
+  val V5_0_0: Version = Version.parse("5.0-alpha1")
```

Comment:
nit - should we use [`5.0-beta1`](https://github.com/apache/cassandra/releases/tag/cassandra-5.0-beta1) here instead?